src/components/forms/constants.ts

```diff
@@ -1434,6 +1454,21 @@ export const HOW_DID_YOU_HEAR_ABOUT_ESP_OPTIONS = [
   { value: 'Other', label: 'Other' }
 ];
 
+export const HOW_DID_YOU_HEAR_ABOUT_GRANTS_WAVE = [
+  { value: 'Email received', label: 'Email received' },
+  ...HOW_DID_YOU_HEAR_ABOUT_ESP_OPTIONS
+];
+
+// academic grants
+export const APPLYING_AS_OPTIONS = [
+  { value: 'An individual', label: 'An individual' },
+  { value: 'A University	', label: 'A University	' },
```

Comment:
Any reason for that space in the value and label?

Reply:
No reason, it's a typo, fixed now!